pallets/difficulty/src/tests.rs

```diff
@@ -118,6 +121,10 @@ fn next_difficulty_should_return_next_when_previous_is_configured() {
 			})
 			.unwrap();
 		assert_eq!(previous.len(), 2);
+		PreviousDifficultiesAndTimestamps::<Test>::put(previous.clone());
+
+		let via_getter = crate::Pallet::<Test>::previous_difficulties_and_timestamps();
+		assert_eq!(via_getter, previous);
```

Comment:
Again just a comment, testing effective reads and writes to the underlying storage seems a bit out-of-scope/misplaced to me. You put `previous` and then immediately read it back and assert on both, I don't see how it helps support the test's premise.